components/autofill/src/error.rs

```diff
@@ -26,11 +26,4 @@ pub enum ErrorKind {
     Utf8Error(#[from] std::str::Utf8Error),
 }
 
-error_support::define_error! {
-    ErrorKind {
-        (SqlError, rusqlite::Error),
-        (IoError, std::io::Error),
-        (InterruptedError, Interrupted),
-        (Utf8Error, std::str::Utf8Error)
-    }
-}
+pub type Result<T> = std::result::Result<T, ErrorKind>;
```

Comment:
Naming nit: now that we don't have the layer of indirection provided by `define_error!`, I wonder if it would be simpler to call this `Error` instead of `ErrorKind`. I don't think it matters too much either way, but it might simplify the API for consumers if they don't have to wonder what an "error kind" is.

Reply:
good idea 👍